Ch05/Ch05.md

```diff
@@ -1018,35 +1007,35 @@ using(var subscription =
 }
 ```
 
-The key parts to the code in Listing 5.17 is the use of `Versioned<Customer>` as the type that we are subscribing on. RavenDB recognize that as a versioned
+The key parts to the code in Listing 5.17 is the use of `Versioned<Customer>` as the type that we're subscribing on. RavenDB recognizes that as a versioned
 subscription and will feed us each and every revision for the `Customer` collection.
 
-Note that the data model, as well, has changed. Instead of getting the document itself, we get an object that has `Previous` and `Current` properties, 
-representing the changes that happened to the document. In other words, you are able to inspect the current and previous versions and make decisions
+Note that the data model as well has changed. Instead of getting the document itself, we get an object that has `Previous` and `Current` properties, 
+representing the changes that happened to the document. In other words, you're able to inspect the current and previous versions and make decisions
 based on the changes that happened to the entity. 
 
-This feature opens up a _lot_ of options regarding analytics because you aren't seeing just a snapshot of the state, but all the intermediate steps along the 
-way. This has utilizations in business analytics, fraud and outliers detection, foresnics and being able to reconstruct the flow of data through your system. 
-The way versioned subscriptions work, we are going to get all the changes that match our criteria, at the same order they happened in the database^[Again, 
-different nodes may have observed the events in a different order, but it should roughly match across the cluster].
+This feature opens up a _lot_ of options regarding analytics because you aren't seeing just a snapshot of the state but all the intermediate steps along the 
+way. This has uses in business analytics, fraud and outlier detection and forensics, in addition to the general benefit of being able to reconstruct the flow of data through your system. 
```

Comment:
Shouldn't this be "fraud  outlier detection and forensics," ?

Reply:
Honestly, I'm not actually sure. I figured you were saying "fraud detection" and "outlier detection." I googled ["fraud outlier detection"](https://www.google.com/search?rlz=1C1CHBF_enUS713US713&q=fraud+outlier+detection&oq=fraud+outlier+d&gs_l=psy-ab.3.0.0i22i30k1l2.370963.383620.0.384974.25.20.3.0.0.0.254.1972.2j13j1.16.0....0...1.1.64.psy-ab..6.19.1986...0j0i131k1j0i67k1j0i20k1j33i160k1.2i7TAv2uRfY) and I don't see any instances on page one of that phrase verbatim, just "outlier detection." I've never heard of "fraud outlier detection," and neither has Erik. And when you search it in quotes, there's almost no quality results with that exact phrase. Take that for what it's worth.

Reply:
Sorry, I meant:  "fraud, outlier detection and forensics,"
I missed a comma :-)